src/app/layout.tsx

```diff
@@ -60,12 +60,11 @@ export default function RootLayout({
         <nav className="relative">
           <div
             onChange={handleScroll}
-            className={`fixed z-20 top-0 left-0 right-0 bg-black ${
-              scrolling ? 'bg-opacity-30' : ''
-            } h-[60px] px-2 md:px-8 py-3 text-white`}
+            className={`fixed z-20 top-0 left-0 right-0 bg-black ${scrolling ? 'bg-opacity-30' : ''
+              } h-[60px] px-2 md:px-8 py-3 text-white`}
```

Comment:
why did you delete the scrolling part?

Reply:
> why did you delete the scrolling part?

It was not deleted. A plugin automatically adjusted the format. I will disable the plugin to maintain consistency with everyone.